src/main/java/io/github/syst3ms/skriptparser/lang/SimpleLiteral.java

```diff
@@ -19,11 +19,12 @@
  */
 @SuppressWarnings("unchecked")
 public class SimpleLiteral<T> implements Literal<T> {
-    private final T[] values;
+
     private boolean isAndList = true;
     private Class<T> returnType;
+    private final T[] values;
 
-    public SimpleLiteral(T[] values) {
+    public SimpleLiteral(T... values) {
```

Comment:
Won't this create issues for SyntaxParser at line 538? Those lines over there should be changed as well. Also, should the other constructor be removed as well or can it make a difference? I'm not experienced enough with Java generics to know if that constructor should exist or not (because it already has a varargs parameter).

Reply:
It shouldn't, but it'll remove that whole Array newInstance. Epic. I will get around to changing that soon.